modules/pixi/PixiLayerRapidOverlay.js

```diff
@@ -0,0 +1,166 @@
+import * as PIXI from 'pixi.js';
+
+import { geojsonFeatures } from '../util/util.js';
+import { AbstractLayer } from './AbstractLayer.js';
+import { PixiFeatureLine } from './PixiFeatureLine.js';
+import { PixiFeaturePoint } from './PixiFeaturePoint.js';
+import { PixiFeaturePolygon } from './PixiFeaturePolygon.js';
+
+
+
+/**
+ * PixiLayerRapidOverlay
+ * This class contains any overlay vectors that should be 'drawn over' the map, usually at low zooms.
+ * The data for these are scraped from the RapidSystem's datasets, specifically the 'overlay' field.
+ * @class
+ */
+export class PixiLayerRapidOverlay extends AbstractLayer {
+
+  /**
+   * @constructor
+   * @param  scene    The Scene that owns this Layer
+   * @param  layerID  Unique string to use for the name of this Layer
+   */
+  constructor(scene, layerID) {
+    super(scene, layerID);
+    this._clear();
+    this._enabled = true;
+
+    const overlays = new PIXI.Container();
+    overlays.name = `${this.layerID}-overlays`;
+    overlays.sortableChildren = false;
+    overlays.interactiveChildren = true;
+    this.overlaysContainer = overlays;
+
+
+    const basemapContainer = this.scene.groups.get('basemap');
+    basemapContainer.addChild(overlays);
+  }
+
+
+  /**
+   * render
+   * Render the GeoJSON custom data
+   * @param  frame      Integer frame being rendered
+   * @param  viewport   Pixi viewport to use for rendering
+   * @param  zoom       Effective zoom to use for rendering
+   */
+  render(frame, viewport, zoom) {
+    if (!this.enabled || !(this.hasData())) return;
+
+    const vtService = this.context.services.vectortile;
+    const datasets = this.context.systems.rapid.datasets;
+
+    // for (const [key, dataset] in datasets) {
+
+    // }
+
+    const customColor = new PIXI.Color(datasets.get('fbRoads').color);
+    const overlay = datasets.get('fbRoads').overlay;
+    let overlayData = [];
+    if (overlay && vtService) {   // fetch data from vector tile service
+      if ((zoom >= overlay.minZoom ) && (zoom <= overlay.maxZoom)) {  // avoid firing off too many API requests 
+        vtService.loadTiles(overlay.url);
+      }
+      overlayData = vtService.getData(overlay.url).map(d => d.geojson);
+    }
+
+    // const polygons = overlayData.filter(d => d.geometry.type === 'Polygon' || d.geometry.type === 'MultiPolygon');
```

Comment:
The vector data we get back seems to consist of two bits of data: 

Polygon borders for a`water` layer (which we can safely discard) 
and Points that have a single property: `weight`. Presumably the `weight` is to control how bright/vivid the heatmap should be, but in empirical testing I saw values from 100 to 300,000.

Reply:
After showing this to Jonah and a few other folks, the existing implementation was fuzzy/blobby enough that they were convinced it was a raster map. I think the existing implementation is more than good enough without needing to resort to darkening/lightening the alpha with a `weight` value interpolation. We can revisit in the future.